packages/pn-personafisica-login/src/services/configuration.service.ts

```diff
@@ -61,8 +61,9 @@ class LoginConfigurationValidator extends Validator<LoginConfigurationFromFile>
 export function getConfiguration(): LoginConfiguration {
   const configurationFromFile = Configuration.get<LoginConfigurationFromFile>();
   const IS_DEVELOP = process.env.NODE_ENV === 'development';
-  const VERSION = process.env.REACT_APP_VERSION ?? '';
-  const BASE_URL = process.env.PUBLIC_URL;
+  const VERSION = import.meta.env.VITE_APP_VERSION ?? '';
+  // const BASE_URL = process.env.PUBLIC_URL;
+  const BASE_URL = '';
```

Comment:
I have unfortunately forgot the details about this. It's clear that `process.env.PUBLIC_URL` cannot be used since such variable is only given a value in CRA, non under Vite. The alternative would be to use `import.meta.env.BASE_URL`. But I wonder whether a non-empty value should be given, since this `BASE_URL` is used only to create paths that are used in a `react-router-dom` router, hence I guess relative paths are OK.
In any case, I tested thoroughly in my locale, both in development mode and from the build (the latter through `vite preview`) and it seems to work.